themes/bootstrap5/js/record.js

```diff
@@ -102,96 +100,96 @@ function refreshRecordRating(recordId, recordSource) {
 }
 
 function registerAjaxCommentRecord(_context) {
-  var context = typeof _context === "undefined" ? document : _context;
+  let context = typeof _context === "undefined" ? document : _context;
   // Form submission
-  $(context).find('form.comment-form').off("submit").on("submit", function commentFormSubmit() {
```

Comment:
Is it a problem that the new code has no equivalent to `off("submit")`? Everything seems to be working correctly, but based on my testing, it looks like a new event listener gets registered every time I switch tabs. Is that okay, or should we do something like use a data element to prevent re-registration?

Reply:
Should be enough to make the event handler a non-inline function. addEventListener won't add the same function multiple times.